pkg/sypgp/sypgp.go

```diff
@@ -57,6 +57,14 @@ type Handle struct {
 	path string
 }
 
+// GenKeyPairOptions parameters needed for generating new key pair.
+type GenKeyPairOptions struct {
+	Name     *string
```

Comment:
Why are you using pointers here?

Reply:
Since i want to distinguish when user didn't pass a flag and when he specified empty field. For example, empty string for password field is a valid value and without reference we will not know whether to ask user input or not.